### PR TITLE
ENH fail if travis not on special arch

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/conda_forge_ci_setup/build_utils.py [hunk 1]
```diff
@@ -84,13 +84,29 @@ def fail_if_outdated_windows_ci(feedstock_root):
...
+    specific_config = safe_load(open(config_file))
+    if "channel_targets" in specific_config:
+        channels = [c.strip().split(" ") for c in specific_config["channel_targets"]]
+    else:
+        update_global_config(feedstock_root)
+        channels = _global_config["channels"]["targets"]
+
+    upload_to_conda_forge = any(owner == "conda-forge" for owner, _ in channels)
+    
+    if upload_to_conda_forge and os.environ.get("CI", None) == "travis" and platform.uname() == "x86_64":
```
Comment (isuruf) [inline recipe/conda_forge_ci_setup/build_utils.py:97]:
```suggestion
    if upload_to_conda_forge and os.environ.get("CI", None) == "travis" and platform.uname().machine in ["x86_64", "AMD64"]:
```

File: recipe/conda_forge_ci_setup/build_utils.py [hunk 2]
```diff
@@ -84,13 +84,29 @@ def fail_if_outdated_windows_ci(feedstock_root):
...
+    specific_config = safe_load(open(config_file))
+    if "channel_targets" in specific_config:
+        channels = [c.strip().split(" ") for c in specific_config["channel_targets"]]
+    else:
+        update_global_config(feedstock_root)
+        channels = _global_config["channels"]["targets"]
+
+    upload_to_conda_forge = any(owner == "conda-forge" for owner, _ in channels)
+    
+    if upload_to_conda_forge and os.environ.get("CI", None) == "travis" and platform.uname().machine in ["x86_64", "AMD64"]:
```
Comment (beckermr) [inline recipe/conda_forge_ci_setup/build_utils.py:97]:
```suggestion
    if upload_to_conda_forge and os.environ.get("CI", None) == "travis" and platform.uname().machine.lower() in ["x86_64", "amd64"]:
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/conda_forge_ci_setup/build_utils.py [hunk 3]
```diff
@@ -84,13 +84,29 @@ def fail_if_outdated_windows_ci(feedstock_root):
...
+    specific_config = safe_load(open(config_file))
+    if "channel_targets" in specific_config:
+        channels = [c.strip().split(" ") for c in specific_config["channel_targets"]]
+    else:
+        update_global_config(feedstock_root)
+        channels = _global_config["channels"]["targets"]
+
+    upload_to_conda_forge = any(owner == "conda-forge" for owner, _ in channels)
+    
+    if upload_to_conda_forge and os.environ.get("CI", None) == "travis" and platform.uname().machine.lower() in ["x86_64", "amd64"]:
```
Comment (beckermr) [inline recipe/conda_forge_ci_setup/build_utils.py:97]:
```suggestion
    if (
        upload_to_conda_forge
        and os.environ.get("CI", None) == "travis"
        and (
            platform.uname().machine.lower() in ["x86_64", "amd64"]
            or platform.system().lower() != "linux"
        )
    ):
```

**Comment by beckermr** [comment]
@isuruf This one is ready for another review!

**Comment by beckermr** [comment]
Any other comments or shall we merge?

**Review by chrisburr — APPROVED** [review]

**Pull request merged**